src/lib/flash-helper.js

```diff
@@ -66,6 +66,12 @@ async function _flashDeviceInNormalMode(device, data, { name, progress, checkSki
 				}
 				return device;
 			}
+			try {
```

Comment:
Makes more sense this approach since prepareToFlash always puts the device in listening for nonDFU modes. 
I was thinking about a final reset after the whole process could be also a good way to have the device back to normal but would be too much since in all the times the device is being resetting when it flashes something hehe.

Reply:
Flashing with control requests resets by itself so it should be in a good state without an extra reset.